docs/tutorials/python/tutorial_scripts/wiki.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python3
+"""
+Tutorial script demonstrating the Synapse Wiki models functionality.
+
+This script shows how to:
+1. Create, read, and update wiki pages
+2. Work with WikiPage Markdown
```

Comment:
Is there another descriptive way we can work some of these actions rather than "Work with.." maybe, describing the create/update/delete/restore actions taking place during that section of code?